### DISCUSSION TITLE
Windows 10 & 11 - location for network settings?

### DISCUSSION THREAD

**Discussion by PC-Support-Services** [rustdesk/rustdesk#9825] [Q&A]
On Windows 10 & 11 machines where does Rustdesk store the network ID, relay, API and key? (file and / or registry)

I had to reinstall my server (pro version) and the public key changed. Remote reinstalling on client machines doesn't allow unattended access. 

I'd like the location(s) so I can delete before reinstalling.

**Comment by PC-Support-Services** [comment]
Anyone?

**Reply by 21pages** [reply]
Install: `C:\Windows\ServiceProfiles\LocalService\AppData\Roaming\RustDesk\config\RustDesk2.toml`
Portable: `%appdata%\RustDesk\config\RustDesk2.toml`

**Comment by PC-Support-Services** [comment]
Many thanks. Working perfectly now.


On 6 Nov 2024, at 11:27, 21pages ***@***.***> wrote:



Install: C:\Windows\ServiceProfiles\LocalService\AppData\Roaming\RustDesk\config\RustDesk2.toml
Portable: %appdata%\RustDesk\config\RustDesk2.toml
